otter/grade/Dockerfile

```diff
@@ -2,15 +2,13 @@ ARG BASE_IMAGE=ucbdsinfra/otter-grader
 FROM ${BASE_IMAGE}
 RUN apt-get update && apt-get install -y curl unzip dos2unix && apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 RUN mkdir -p /autograder/source
-ARG ZIPPATH
 ARG BASE_IMAGE
-RUN echo $BASE_IMAGE
-ADD ${ZIPPATH} /tmp/autograder.zip
 ENV BASE_IMAGE=$BASE_IMAGE
-RUN unzip -d /autograder/source /tmp/autograder.zip && \
-    cp /autograder/source/run_autograder /autograder/run_autograder && \
-    dos2unix /autograder/run_autograder /autograder/source/setup.sh && \
+ADD run_autograder /autograder/run_autograder
+ADD setup.sh environment.yml requirements.* /autograder/source/
+RUN dos2unix /autograder/run_autograder /autograder/source/setup.sh && \
     chmod +x /autograder/run_autograder && \
     apt-get update && bash /autograder/source/setup.sh && apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* && \
     mkdir -p /autograder/submission && \
     mkdir -p /autograder/results
+ADD . /autograder/source
```

Comment:
What is the purpose of this addition? I can see this causing issues e.g. with conflicting filenames.

Reply:
Conflicting filenames should not cause more problems than before. The 'trick' I want to apply is to copy the (probably) similar files in the step before and afterwards all remaiming files. The files should still only be the content of the autograder.zip